main.py

```
@@ -0,0 +1,54 @@
+logger.info(f"Test dataset: {missing_dates_test_message}")
+
+duplicate_train_message = data_checker.unique_id_per_date_check(train_df)
+logger.info(f"Train dataset: {duplicate_train_message}")
+
+duplicate_test_message = data_checker.unique_id_per_date_check(test_df)
+logger.info(f"Test dataset: {duplicate_test_message}")
+
+data_quality_message = data_checker.nulls_and_shape_check()
+logger.info(data_quality_message)
+
+logger.info("Data checking completed.")
```

Ruff

No newline at end of file

Add trailing newline

(W292)